lib/utils.js

```diff
@@ -80,6 +68,20 @@ const getFragmentAssetsToPreload = (headers, { host } = {}) => {
     return assetsToPreload;
 };
 
+function getFragmentAssetUris(refs, assetSize) {
```

Comment:
Why put this in utils, is it used anywhere else?

Are we planning to add this to the exported API?